### PR TITLE
Pl/layout fix

### PR DESCRIPTION
**Pull request by lizonp**
screen shot 2018-03-12 at 10 00 29
screen shot 2018-03-12 at 10 00 18

### CODE REVIEW

File: webpack/webpack.config.dev.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = merge(configCommon, {
         pathRewrite: { "^/node": "" },
       },
       "/api/signature": {
-        target: "http://localhost:5000",
```
Comment (huesforalice) [inline webpack/webpack.config.dev.js]: why is this changed? It should stay at 5000, afaik there have been no changes here.

File: app/styles/global.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  display: flex;
+  flex-direction: column;
+  min-height: calc(100% - #{$header-height});
+}
+
+.layout-container {
+  @include media-breakpoint-up (md) {
+    padding-left: 30px;
```
Comment (huesforalice) [inline app/styles/global.scss]: if you use pixels, please use px-to-rem, so 30px becomes px-to-rem(30px)

Comment (huesforalice) [inline app/styles/global.scss]: In general you can use bootstrap classes for padding and spacing, please see this documentation: https://getbootstrap.com/docs/4.0/utilities/spacing/

File: app/components/dashboard/notification-widget/Notification.module.scss [hunk 3]
```diff
@@ -36,6 +31,10 @@
     }
   }
 
+  + #{&} {
```
Comment (Moejoe90) [inline app/components/dashboard/notification-widget/Notification.module.scss]: why is `id` used here?

File: app/styles/global.scss (hunk 2 above)
Comment (lizonp) [reply app/styles/global.scss]: It's changed.

File: app/components/dashboard/notification-widget/Notification.module.scss (hunk 3 above)
Comment (lizonp) [reply app/components/dashboard/notification-widget/Notification.module.scss]: It's sass interpolation. So it's the same as `.notification + .notification`

File: webpack/webpack.config.dev.js (hunk 1 above)
Comment (lizonp) [reply webpack/webpack.config.dev.js]: I have commited it by mistake, it's changed now.

**Comment by Moejoe90** [comment]
btw i think you can use `w-100` for the same exact thing

File: app/styles/global.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  @include media-breakpoint-up (md) {
+    padding-left: px-to-rem(30px);
+    padding-right: px-to-rem(30px);
+    padding-top: px-to-rem(20px);
+  }
+
+  max-width: 100%;
+  min-height: 100%;
+  overflow-x: hidden;
+  overflow-y: scroll;
```
Comment (banciur) [inline app/styles/global.scss]: Do we need to be like this? Now I get double scroll bars one that is inactive. 
scroll

File: app/styles/global.scss [hunk 5]
```diff
@@ -0,0 +1,34 @@
+html,
```
[the remark this answers is not in the document]
Comment (lizonp) [reply app/styles/global.scss]: I's changed 👍

File: app/components/layouts/LayoutAuthorized.tsx [hunk 6]
```diff
@@ -6,12 +6,14 @@ import { LayoutAuthorizedMenu } from "./LayoutAuthorizedMenu";
...
-  <Row className="h-100">
-    <Col xs={12} sm="auto">
-      <div className={styles.menu}>
-        <LayoutAuthorizedMenu />
-      </div>
-    </Col>
-    <Col className={styles.content}>{children}</Col>
-  </Row>
+  <>
+    <div className={styles.menu}>
```
[the remark this answers is not in the document]
Comment (lizonp) [reply app/components/layouts/LayoutAuthorized.tsx]: I's changed 👍

[the remark this answers is not in the document]
Comment (banciur) [reply app/components/layouts/LayoutAuthorized.tsx]: It's still here - https://github.com/Neufund/platform-frontend/blob/pl/layout-fix/app/components/layouts/LayoutAuthorized.tsx#L10
This line instead `<div className={styles.menu}>` should be just `<div>`

[the remark this answers is not in the document]
Comment (lizonp) [reply app/components/layouts/LayoutAuthorized.tsx]: @banciur just fixed it

File: app/styles/global.scss [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+body,
+[id="app"] {
+  height: 100%;
+  min-height: 100%;
+}
+
+.wrapper {
```
Comment (krzkaczor) [inline app/styles/global.scss]: okay we will figure today if we need this or we go with row/column or something else. for now it stays
